doc/release-notes/release-8.sgml.in

```diff
@@ -157,6 +157,15 @@ This section gives an account of those changes in three categories:
 	<p>Removed <em>LM_Group</em> helper. The LM protocol is
 	   insecure and no longer supported on the market since 2008.
 
+	<tag>--disable-arch-native</tag>
+	<p>The <em>-march=native</em> compiler option is not used by
+		default. It is possible to enable it by using the
+		<em>--enable-arch-native</em> option instead.
+		This optimization may cause problems when Squid is
```

Comment:
After other edits removed text that described this compiler option as an optimization, a causal reader may not understand what "optimization" this text is talking about (e.g., the new default, the old default, the explicit option, or something else):

```suggestion
		Using <em>-march=native</em> may cause problems when Squid is
```